src/loops/programs_with_repetition/hard/q7/SinOfTheTimes.java

```diff
@@ -0,0 +1,31 @@
+package loops.programs_with_repetition.hard.q7;
+
+/*
+Write a program called SinOfTheTimes that will calculate an approximation of sin(x) from n = 0 until n = 10, with x = 3.14,
```

Comment:
I think we should change this question to specify one of the two numbers, so they can't hardcode it.

Reply:
And then we can vary the other one on the website

Reply:
Yeah, but if we change the values, they could still just use something like wolfram alpha to quickly calculate the series, and then just print that out.